roles/openshift_deploy_registry/tasks/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Deploy registry
+  shell: osadm registry --create --credentials=/var/lib/openshift/openshift.local.certificates/openshift-registry/.kubeconfig --images='registry.access.redhat.com/openshift3_beta/ose-${component}:${version}'
```

Comment:
We should probably test if the registry we are creating already exists (technically, we could have multiple registries, so we should probably provide a default label and test for existence based on that label).